kube_dns/README.md

```diff
@@ -0,0 +1,35 @@
+# Kube-dns Integration
+
+## Overview
+
+Get metrics from kube-dns service in real time to:
+
+* Visualize and monitor dns metrics collected via Kubernetes' kube-dns addon
+  through Prometheus
+
+See https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns for
+more informations about kube-dns
+
+## Installation
+
+Install the `dd-check-kube_dns` package manually or with your favorite configuration manager
+
+## Configuration
+
+Edit the `prometheus.yaml` file to point to your server and port, set the masters to monitor
```

Comment:
Oops, good catch indeed. Sorry for that.
Commit amended, PR updated.
Thanks! :)